src/pharmpy/model/execution_steps.py

```diff
@@ -153,8 +154,7 @@ def __init__(
         solver_rtol: Optional[int] = None,
         solver_atol: Optional[int] = None,
         tool_options: Optional[frozenmapping[str, Any]] = None,
-        eta_derivatives: Optional[tuple[str, ...]] = None,
-        epsilon_derivatives: Optional[tuple[str, ...]] = None,
+        derivatives: tuple[tuple[Type[Expr]], ...] = (),
```

Comment:
It should be `Expr` rather than `Type[Expr]`

Reply:
Changed all types to `Expr`